conbench/templates/benchmark-result.html

```diff
@@ -126,9 +127,7 @@
       {% for k,v in benchmark.tags.items() %}
       <li class="list-group-item" style="overflow-y: auto;">
         <b>{{ k }}</b>
-        {% if v is not none %}
         <div align="right" style="display:inline-block; float: right;">{{ v }}</div>
-        {% endif %}
```

Comment:
This is just to make sure we show what's in the database. a `param=None` as of today contributes defining a specific case permutation.